packages/contentstack-import/src/import/module-importer.ts

```diff
@@ -23,6 +23,8 @@ class ModuleImporter {
   }
 
   async start(): Promise<any> {
+    const stackName: Record<string, unknown> = await this.stackAPIClient.fetch();
+    this.importConfig.stackName = stackName.name as string;
```

Comment:
@cs-raj Can you validate this if it works fine with the management token?

Reply:
Validated, the code threw an error, hence removed the condition to use stack name if management token is used